test/integration-tests/Makefile

```diff
@@ -29,3 +29,11 @@ run-integration-tests: build-tests mock-image vpp-image
 	@echo "Running Integration tests..."
 	@echo "Running Calico VPP Agent - CNI tests..."
 	${SUDO} env "PATH=$$PATH" VPP_BINARY=/usr/bin/vpp INTEGRATION_TEST=. VPP_IMAGE="${VPP_IMAGE}" ./cni.test -test.v -test.run Integration
+
+VPP_DEV_DIR ?= /repo/vpp-manager/vpp_build/build-root/install-vpp_debug-native
+dev: build-tests mock-image
```

Comment:
I find a bit sad to define another target instead of augmenting the existing one...sigh
But up to you